servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/DefaultHttpLoadBalancerFactory.java

```diff
@@ -80,6 +113,31 @@ public FilterableStreamingHttpLoadBalancedConnection toLoadBalancedConnection(
         return new DefaultFilterableStreamingHttpLoadBalancedConnection(connection);
     }
 
+    @Override
+    public FilterableStreamingHttpLoadBalancedConnection toLoadBalancedConnection(
+            final FilterableStreamingHttpConnection connection,
+            final ReservableRequestConcurrencyController concurrencyController,
+            @Nullable final ContextMap context) {
+
+        RequestTracker hostHealthIndicator = null;
+        if (context == null) {
+            LOGGER.debug("Context is null. In order for " + DefaultHttpLoadBalancerFactory.class.getSimpleName() +
+                    ":toLoadBalancedConnection to get access to the " + RequestTracker.class.getSimpleName() +
+                    ", health-monitor of this connection, the context must not be null.");
+        } else {
+            hostHealthIndicator = context.get(REQUEST_TRACKER_KEY);
+            if (hostHealthIndicator == null) {
+                LOGGER.debug(REQUEST_TRACKER_KEY.name() + " is not set in context. " +
+                        "In order for " + DefaultHttpLoadBalancerFactory.class.getSimpleName() +
+                        ":toLoadBalancedConnection to get access to the " + RequestTracker.class.getSimpleName() +
+                        ", health-monitor of this connection, the context must be properly wired.");
+            }
+        }
+
+        return new DefaultHttpLoadBalancedConnection(connection, concurrencyController,
+                errorClassFunction, peerResponseErrorClassifier, hostHealthIndicator);
```

Comment:
When there are no hostHealthIndicator, we should use `HttpLoadBalancerFactory.DefaultFilterableStreamingHttpLoadBalancedConnection` to keep the existing path as-is.